packages/fuels-contract/src/contract.rs

```diff
@@ -306,7 +308,9 @@ impl Contract {
         if simulate {
             return script.simulate(fuel_client).await;
         }
-        script.call(fuel_client).await
+        let receipts = script.call(fuel_client).await;
+        event!(Level::DEBUG, "{:?}", receipts);
```

Comment:
isn't this the same thing as `tracing::debug!`?

Reply:
No it's not the same, it's better than what I used. I didn't know about this approach so I will replace event () with debug ! :)